docs/processes/github_security_bypass.md

```diff
@@ -0,0 +1,39 @@
+# Bypassing GitHub security controls
+
+We have controls on our [Defra GitHub organisation](https://github.com/defra) that require approval to:
+- bypass GitHub push protection
+- dismiss GitHub security alerts
+
+This guide covers how these processes work and how to request approval.
+
+## Push protection
+
+All of our public repositories have [GitHub secret scanning](https://docs.github.com/en/enterprise-cloud@latest/code-security/secret-scanning/introduction/about-secret-scanning) enabled, with [push protection](https://docs.github.com/en/enterprise-cloud@latest/code-security/secret-scanning/introduction/about-push-protection) enforced.
+
+This means that you will not be able to push code up to a public repository if GitHub detects a secret in your code.
+
+Secret scanning is pretty good at avoiding false-positives, but occasionally this may happen. When it does, you'll need to bypass the push protection, which you can do through the GitHub UI, by providing a reason for the bypass.
+
+When you do this, it will create a request that requires approval - **you will need to post a message in the #github-support channel on the defra-digital Slack to have your request actioned**
+
+You should provide:
+- a link to the bypass request
+- a brief explanation of why the protection should be bypassed
```

Comment:
Could we do a numbered list as opposed to bullets with a and b?

Reply:
I could make it a numbered list, but it's not a and b at the moment, I didn't think it really needed numbering as there's only two items?